packages/SwingSet/test/test-gc-kernel.js

```diff
@@ -1188,12 +1188,13 @@ test('device transfer', async t => {
   function getRefCounts() {
     return kvStore.get(`${kref}.refCount`); // e.g. "1,1"
   }
+  // XXX TODO: this comment should change, but I'm not 100% sure what the new text should say
```

Comment:
"The device should hold a reachable+recognizable reference, and vat-left (which forgot about amy) does not contribute either form of refcount, making the expected count 1,1. If deviceKeeper..."